### PR TITLE
Add fun clear field

### PR DESCRIPTION
**Pull request by KuzminVik**

### CODE REVIEW

File: fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/livedata/BaseLiveDataFormField.kt [hunk 1]
```diff
@@ -55,6 +55,11 @@ abstract class BaseLiveDataFormField<D, E>(
         showValidationError.value = true
         return isValid.value
     }
+
+    override fun clear() {
+        setValue(initialValue)
+        showValidationError.value = false
+    }
```
Comment (kramlex) [inline fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/livedata/BaseLiveDataFormField.kt:62]: maybe you should also add the reset validation function?

Comment (kramlex) [reply fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/livedata/BaseLiveDataFormField.kt:62]:
```

Comment (kramlex) [reply fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/livedata/BaseLiveDataFormField.kt:62]:
```kotlin
fun resetValidation() {
    showValidationError.value = false
}
```

Comment (KuzminVik) [reply fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/livedata/BaseLiveDataFormField.kt:62]: yes, the function of resetting validation without deleting the value can also be useful

File: fields-flow/src/commonMain/kotlin/dev/icerock/moko/fields/flow/BaseFlowFormField.kt [hunk 2]
```diff
@@ -65,6 +65,15 @@ abstract class BaseFlowFormField<D, E>(
         showValidationError.value = true
         return isValid.value
     }
+
+    override fun clear() {
```
Comment (Alex009) [inline fields-flow/src/commonMain/kotlin/dev/icerock/moko/fields/flow/BaseFlowFormField.kt:69]: i think we should remove `clear` fun - anybody can implement this by self just set data value and call `resetValidation`.
but currenct `clear` implementation have unexpected logic - not remove all data from input but set `initialValue`, that not expected for name `clear`

**Review by Alex009 — CHANGES_REQUESTED** [review]

**Comment by ExNDY** [comment]
Hello, @Alex009, when we can get this update?

**Review by Alex009 — APPROVED** [review]

**Pull request merged**